src/components/Shared/Buttons/ButtonIcon.tsx

```diff
@@ -0,0 +1,77 @@
+import styled, { css } from 'styled-components';
+import { Button } from 'components/Shared/Buttons/Button';
+import ManagedIcon from 'assets/icon/helpers/ManagedIcon';
+
+interface ButtonIconProps {
+  nameIcon: string;
```

Comment:
dodałam w ManageIcon 

```
type Icons =
  | 'microphone'
  | 'check'
  | 'speaker'
  | 'logout'
  | 'back'
  | 'next'
  | 'xmark'
  | 'translate'
  | 'add'
  | 'chat';
```

wyeksportuj to sobie i pobierz tutaj. i zrób żeby nameIcon mogło być tylko tego typu. Lepiej to nam zabezpieczy ikony